src/collisions_updater.cpp

```diff
@@ -0,0 +1,228 @@
+#include <ros/ros.h>
+#include <ros/package.h> // for getting file path for loadng images
+
+#include <moveit/setup_assistant/tools/compute_default_collisions.h>
+#include <moveit/setup_assistant/tools/moveit_config_data.h>
+
+#include <boost/filesystem.hpp>  // for reading folders/files
+#include <boost/program_options.hpp>
+#include <boost/algorithm/string/join.hpp>
+
+#include <fstream>  // for reading in urdf
+#include <streambuf>
+
+namespace fs = boost::filesystem;
+namespace po = boost::program_options;
+
+bool loadFileToString(std::string& buffer, const std::string& path, const std::vector<std::string> &xacro_args){
+
+  if(path.find(".xacro") != std::string::npos){
+        std::string cmd = "rosrun xacro xacro ";
```

Comment:
As you already know I do not like the ROS C++ style, it is harder to read.
However, I can run an automatic formatter before the merge.